PowershellTools.HostService/ServiceManagement/PowershellIntelliSenseService.cs

```diff
@@ -29,11 +31,22 @@ public PowershellIntelliSenseService()
         /// <param name="caretPosition">The caret position.</param>
         /// <returns>A completion results list.</returns>
         public CompletionResultList GetCompletionResults(string script, int caretPosition)
-        {           
+        {
             var commandCompletion = CommandCompletionHelper.GetCommandCompletionList(script, caretPosition, _runspace);
             return CompletionResultList.FromCommandCompletion(commandCompletion);
         }
 
+        public ParseErrorItem[] GetParseErrors(string spanText)

```

Comment:
Do we have tests around the Error Tagging that specifically target the outofproc scenario?